### DISCUSSION TITLE
Timeline CSS not being applied

### DISCUSSION THREAD

**Discussion by HadiSDev** [saadeghi/daisyui#2840] [Q&A] [answered]
Hi

I am trying to use the new timeline component, but it seems to me that the styling is not being applied probably:
https://stackblitz.com/edit/daisyui-astro-tw4qcf?file=src%2Fpages%2Findex.astro

**Comment by NikarashiHatsu** [comment, marked as answer]
It looks like the main issue here is that the DaisyUI version you're using (3.3.3) doesn't have the timeline component that you're trying to use. But don't worry! You can solve this problem by upgrading to the latest version (4.6.2), which includes the timeline component. That should fix the issue you're experiencing.

<img width="1680" alt="Screenshot 2024-02-11 at 04 30 12" src="https://github.com/saadeghi/daisyui/assets/43662951/b0dd3903-596b-4ac9-9154-a3fcb717dab7">
